mongonaut/views.py

```diff
@@ -101,7 +101,7 @@ def get_queryset(self):
             self.page = 1
 
         obj_count = queryset.count()
-        self.total_pages = math.ceil(obj_count / self.documents_per_page)
+        self.total_pages = math.ceil(float(obj_count) / self.documents_per_page)
```

Comment:
Why does casting to a `float` fix this issue? I don't understand how float casting fixes a negative index issue.

Reply:
Ensuring `self.page` is always 1 or greater will be easier to understand for future readers of this code. 

The current fix does not help a future reader of the code to understand why the value is cast to a `float`. If I came to this code in 3 months or later, I would likely remove the `float` cast thinking it caused no problems. I would, however, be wrong and regress a the issue you are fixing.

Reply:
Makes sense. 
Updated.